Dockerfile

```diff
@@ -3,8 +3,10 @@
 FROM node:16-alpine
```

Comment:
We should probably bump this to 18 while we are touching this repository, but that can be done as a separate PR